Shared/Samples/Create symbol styles from web styles/CreateSymbolStylesFromWebStylesView.swift

```diff
@@ -0,0 +1,278 @@
+
+import ArcGIS
+import SwiftUI
+
+struct CreateSymbolStylesFromWebStylesView: View {
+    /// The display scale of the environment.
+    @Environment(\.displayScale) private var displayScale
+    
+    /// A map with a light gray basemap centered on LA County.
+    @State private var map: Map = {
+        let map = Map(basemapStyle: .arcGISLightGray)
+        map.referenceScale = 1e5
+        map.initialViewpoint = Viewpoint(
+            latitude: 34.28301,
+            longitude: -118.44186,
+            scale: 1e4
+        )
+        return map
+    }()
+    
+    /// A feature layer with the LA County Points of Interest service.
+    @State private var featureLayer: FeatureLayer = {
+        let featureTable = ServiceFeatureTable(url: .laPointsOfInterest)
+        return FeatureLayer(featureTable: featureTable)
+    }()
+    
+    /// The legend items for the different symbols from the selected symbol style.
+    @State private var legendItems: [LegendItem] = []
+    
+    /// A Boolean value indicating whether the legend sheet is showing.
+    @State private var isShowingLegend = false
+    
+    var body: some View {
+        MapView(map: map)
+            .onScaleChanged { scale in
+                // Prevent the symbols from scaling when the map zooms out too far.
+                featureLayer.scalesSymbols = scale >= 8e4
+            }
+            .task(id: displayScale) {
+                // Update the symbols when the display scale changes.
+                await updateSymbols(scale: displayScale)
+            }
+            .toolbar {
+                ToolbarItem(placement: .bottomBar) {
+                    Button("Legend") {
+                        isShowingLegend = true
+                    }
+                    .sheet(isPresented: $isShowingLegend, detents: [.medium]) {
+                        legendList
+                    }
+                }
+            }
+    }
+    
+    /// The legend list that describes what each symbol represents.
+    private var legendList: some View {
+        NavigationView {
+            List(legendItems, id: \.name) { legendItem in
+                Label {
+                    Text(legendItem.name)
+                } icon: {
+                    Image(uiImage: legendItem.image)
+                }
+            }
+            .navigationTitle("Symbol Styles")
+            .navigationBarTitleDisplayMode(.inline)
+            .toolbar {
+                ToolbarItem(placement: .confirmationAction) {
+                    Button("Done") {
+                        isShowingLegend = false
+                    }
+                }
+            }
+        }
+        .navigationViewStyle(.stack)
+        .frame(idealWidth: 320, idealHeight: 428)
+    }
+}
+
+private extension CreateSymbolStylesFromWebStylesView {
+    /// Updates the symbols using the symbol style.
+    /// - Parameter scale: The display scale for the swatch images.
+    private func updateSymbols(scale: CGFloat) async {
```

Comment:
```suggestion
    private func updateSymbols(displayScale: CGFloat) async {
```
Because the sample used map scale and display scale, I think it would be a good idea to use `displayScale` in this method.